### PR TITLE
Add enchanced watermark filter

### PR DESCRIPTION
**Pull request by moser4035**
Added missing property to watermarkfilter and textfilter
Added signaling support to he aac codec configurations

### CODE REVIEW

File: tests/bitmovin/services/codecconfigurations/he_aac_v2_codec_configurations_tests.py [hunk 1]
```diff
@@ -127,22 +127,26 @@ def _compare_he_v2_aac_codec_configurations(self, first: HeAACv2CodecConfigurati
...
 
     def _get_sample_he_v2_aac_codec_configuration(self):
-        he_aac_v2_codec_configuration = HeAACv2CodecConfiguration(name='Python - Sample AAC Codec Configuration',
-                                                                  description='More declarative description for CodecConfig',
-                                                                  bitrate=128000,
-                                                                  rate=48000,
-                                                                  volume_adjust=None,
-                                                                  normalize=None,
-                                                                  channel_layout=AACChannelLayout.CL_2_1)
+        he_aac_v2_codec_configuration = HeAACv2CodecConfiguration(
```
Comment (dominic-miglar) [inline tests/bitmovin/services/codecconfigurations/he_aac_v2_codec_configurations_tests.py]: also write a test where signaling is set to None
